playwright/tests/pageobjects/abstract/Table.ts

```diff
@@ -15,13 +17,34 @@ export class Table {
     return new Row(this.rows, index);
   }
 
+  async focusTable() {
+    await this.table.focus();
+  }
+
   async expectToHaveValues(...values: Array<Array<string>>) {
     await expect(this.rows).toHaveCount(values.length);
     for (let i = 0; i < values.length; i++) {
       await this.row(i).expectToHaveValues(...values[i]);
     }
   }
 
+  async expectToBeSelected(indexes: Array<number>) {

```

Comment:
Use the spread syntax and you can omit the square bracket (`[]`) when calling this method:
```typescript
async expectToBeSelected(...indexes: Array<number>)
```

can be called like ths
```typescript
await editor.table.expectToBeSelected(0, 1, 2);
```